jung-visualization/src/main/java/edu/uci/ics/jung/visualization/layout/AWT.java

```diff
@@ -0,0 +1,14 @@
+package edu.uci.ics.jung.visualization.layout;
+
+import edu.uci.ics.jung.layout.model.PointModel;
+import java.awt.geom.Point2D;
+
+/**
+ * A home for singleton models for AWT rendering geometry
+ *
+ * @author Tom Nelson
+ */
+public interface AWT {
+
+  PointModel<Point2D> POINT_MODEL = new AWTPointModel();
```

Comment:
static final

Reply:
Last time I checked, every member declared in an interface is static and final.

I usually continue to preach: 'To declare it as such implies that it can be otherwise and so it is an error'

Reply:
Fair point.  I'd forgotten about that aspect of the spec, I don't use interface members much at all.